video.js

```diff
@@ -419,8 +421,11 @@ define(['./teletext', './utils'], function (Teletext, utils) {
                 // The 6845 in the BBC will notice this equality on any clock
                 // tick, not just at the end of a character row.
                 // Required by Wave Runner.
-                if (this.vertCounter === this.regs[6]) {
+                if (this.vertCounter === this.regs[6] && (this.dispEnabled & VDISPENABLE)) {
                     this.dispEnabled &= ~VDISPENABLE;
+                    // Perhaps surprisingly, this happens here. Both cursor
+                    // blink and interlace cease if R6 > R4.
+                    this.frameCount++;
```

Comment:
Just checking...this will wedge after (2**53 / 50) seconds... that's 5,712,328 years. We should be ok :D (but I had to check...the cpucycles counter used to suffer and that _did_ eventually break...)